sktime/transformers/series_as_features/dictionary_based/__init__.py

```diff
@@ -4,6 +4,6 @@
     "SAX"
 ]
 
-from sktime.transformers.series_as_features.dictionary_based.PAA import PAA
-from sktime.transformers.series_as_features.dictionary_based.SAX import SAX
-from sktime.transformers.series_as_features.dictionary_based.SFA import SFA
+from sktime.transformers.series_as_features.dictionary_based.paa import PAA
```

Comment:
Could you make the files private, e.g. rename `.paa` to `._paa`, so that users only import via `.dictionary_based`?